src/styles/card.css

```diff
@@ -0,0 +1,94 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  padding: 32px 16px 0;
+  border-radius: 5px;
+  border: 1px solid var(--light-white);
+  width: var(--card-width);
+  background-color: var(--white);
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+  color: var(--dark-blue);
+}
+
+.card__product-code {
+  margin-bottom: 16px;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--grey);
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-bottom: 24px;
+}
+
+.card__reviews {
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--dark-blue);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+
+  margin-bottom: 16px;
+}
+
+.card__price-text {
+  font-size: 12px;
+  font-weight: 400;
+  font-style: normal;
+  line-height: 18px;
+  color: var(--grey);
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  font-style: normal;
+  line-height: 18px;
+  color: var(--dark-blue);
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  text-decoration: none;
+  align-items: center;
+  text-transform: uppercase;
+  outline: none;
+
+  height: 40px;
+  line-height: 16px;
+  font-size: 14px;
+  font-weight: 700;
+  margin-bottom: 16px;
+  border-radius: 5px;
+  border: 1px transparent;
```

Comment:
as I understand, we should make border 1px solid transparent and then in :hover change it like: border-color: value;

Reply:
it does not work in this way

Reply:
i dont agree with you

Reply:
border: 1px solid transparent;